recipes-openxt/qemu-dm/qemu-dm-2.6.2/block-remove-unused-block-format-support.patch

```diff
@@ -0,0 +1,79 @@
+From 5e9813ea9c6794307937af2c8bf709b99e065468 Mon Sep 17 00:00:00 2001
```

Comment:
These are new patches, please use the standard patch comment block that all the other patches use.